spec/controllers/brapi/v1/studies_controller_spec.rb

```diff
@@ -0,0 +1,167 @@
+require 'rails_helper'
+require 'pp'
+
+RSpec.describe Brapi::V1::StudiesController do
```

Comment:
One question... I'm starting with the request spec and I would like to execute it alone, but an error appears:
rspec spec/requests/brapi/v1/brapi_spec.rb 
/Users/horroc/Documents/Proyects GIT/brassica-own-fork/brassica/spec/rails_helper.rb:63:in `block in <top (required)>': uninitialized constant RSpecHtmlMatchers (NameError) ...

Is it possible to execute only one rspec?